schema.graphql

```diff
@@ -254,47 +255,53 @@ type Scope @jsonField {
 
 type ClaimScope @entity {
   id: ID!
-  targetDid: String! @index
+  targetDid: String! @index(unique: false)
   ticker: String
   scope: Scope
 }
 
 type Claim @entity {
   id: ID!
-  targetDid: String! @index
-  issuer: String! @index
-  issuanceDate: BigInt!
-  lastUpdateDate: BigInt!
-  expiry: BigInt
-  filterExpiry: BigInt! @index
-  type: String! @index
-  jurisdiction: String
-  scope: Scope
-  cddId: String
-}
-
-type ClaimJson @jsonField {
-  targetDid: String!
-  issuer: String!
+  block: Block! @index(unique: false)
+  eventIdx: Int! @index(unique: false)
+  targetDid: IdentityWithClaims! @index(unique: false)
```

Comment:
```suggestion
  target: IdentityWithClaims! @index(unique: false)
```
This is given us `targetDidId`, which is a weird doubling of "Id". DID is the ID and now its a reference to an Entity.